desktop-src/LearnWin32/windows-coding-conventions.md

```diff
@@ -116,12 +104,13 @@ In its original form, Hungarian notation gives *semantic* information about a va
 
 A more common form of Hungarian notation uses prefixes to give *type* information—for example, *dw* for **DWORD** and *w* for **WORD**.
 
-If you search the Web for "Hungarian notation," you will find a lot of opinions about whether Hungarian notation is good or bad. Some programmers have an intense dislike for Hungarian notation. Others find it helpful. Regardless, many of the code examples on MSDN use Hungarian notation, but you don't need to memorize the prefixes to understand the code.
+Note, the cpp core guildines discourage Hungarian notation, see [NL.5: Avoid encoding type information in names](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#nl5-avoid-encoding-type-information-in-names), and
```

Comment:
Suggest "prefix notation" rather than "Hungarian notation".

Reply:
It is known as both.  Do you just want to ignore the reference to Hungarian?  I am not sure you need to as things such as Reverse Polish Notation have history.

Reply:
how about "prefix notation (e.g. Hungarian)" ?